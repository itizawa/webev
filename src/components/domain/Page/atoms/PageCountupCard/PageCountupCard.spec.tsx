import renderer from 'react-test-renderer';
import { PageCountupCard } from './index';

test('PageCountupCard', () => {
  const tree = renderer.create(<PageCountupCard count={100} text="Total Page" />).toJSON();

  expect(tree).toMatchSnapshot();
});
