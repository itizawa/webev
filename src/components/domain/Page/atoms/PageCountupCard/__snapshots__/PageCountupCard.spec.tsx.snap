// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageCountupCard 1`] = `
<div
  className="card bg-dark border border-warning text-white p-3 text-center"
>
  <h2>
    <span />
     Pages
  </h2>
  <p>
    Total Page
  </p>
</div>
`;
